simpleviewinc/tap-resolver
# Add svkeg repos

## Context

* Github NPM packages are a pain to work with
* They always require a git key
* They require a .npmrc to overwrite the default settings

## Goal

* Update the repo to be deployed to npm
  * https://www.npmjs.com/package/@svkeg/tap-resolver
* Use the `@svkeg` organization
* Updated the dependencies to use the `@svkeg` org
  * See https://www.npmjs.com/search?q=%40svkeg

## Updates

* `.npmrc`
  * Removed
* `package.json`
  * Updated to be deployed to npm
* `...all other files`
  * Updated to use `@svkeg/jsutils`

## Testing

* Pull down branch
  * `cd /path/to/tap-resolver && keg pr 23`
* Install all deps
  * `yarn install`
  * Ensure all deps install properly. There should be no errors
* Run tests
  * `yarn test`
  * Ensure all tests pass
